var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -74,6 +74,7 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
         "hwloc": [False, "Whether to enable the HWLOC library"],
         "numactl": [False, "Whether to enable the LIBNUMA library"],
         "memkind": [False, "Whether to enable the MEMKIND library"],
+        "rocthrust": [False, "Whether to enable the thrust library for the HIP backend"],
```

Comment:
```suggestion
```

After more thought, always enabling `rocthrust` when enabling `room` makes more sense for the users. And it is more straightforward.